.github/workflows/pr-code-checks-directory-size-exporter.yml

```diff
@@ -0,0 +1,34 @@
+name: PR Code Checks Directory Size Exporter
```

Comment:
Why is this workflow needed?
I don't see the `dependencies` directory being ignored her https://github.com/kyma-project/telemetry-manager/pull/2428/files#diff-1a107600c2fae56cabc1a652104deec569328a816e22689cbe4bf5f017ddd368R12